### PR TITLE
evm docs improvements

### PR DESCRIPTION
**Pull request by chasefleming**

### CODE REVIEW

File: docs/evm/Build/accounts.md
Comment (franklywatson) [inline docs/evm/Build/accounts.md:1]: I have a big update to this which I haven't yet committed. Are you ok to revert this?

File: docs/evm/about.md
Comment (franklywatson) [inline docs/evm/about.md:1]: Also a big update to this one the whole current paragraph will be replaced

File: docs/evm/using.md
Comment (franklywatson) [inline docs/evm/using.md:1]: Same here

**Review by JeffreyDoyle — APPROVED** [review]

File: docs/evm/Build/accounts.md
Comment (chasefleming) [reply docs/evm/Build/accounts.md:1]: Feel free to change whatever you think

**Pull request merged**